Leaflet.Graticule.js

```diff
@@ -192,7 +200,17 @@ L.LatLngGraticule = L.Layer.extend({
             return this.options.latFormatTickLabel(lat);
         }
 
+
```

Comment:
Extra line here... but won't you have to do this to `__format_lon` too? 

Also, I'm not sure you want to mix in the `truncateToPrecision` code here, since that logic may not hold if others don't use the same relationship between precision and zoom. Can't you assess the precision of the passed `lat` and just truncate to that, instead of relating it to zoom? Remember that the Graticule plugin doesn't make the assumptions our library does -- it's configurable. So could you assess the precision of, say, `0.001` with something like:

```js
var decimalPlacesAfterZero = 0;
if (lat < 1) placesAfter = (lat + '').split('.')[1].length;
```
And then just truncate it right at the lines that used to be:

```js
return '' + lng;
```